site/index.html

```diff
@@ -0,0 +1,163 @@
+      <h4>Festivals in Nepal</h4>
+      <p>Nepal is a culturally diverse country, and its festivals reflect the various ethnic backgrounds and rich religious practices of its people. Religious ceremonies, bright processions, music, dance, and social events are all common features of Nepali festivals, which are widely celebrated with intense zeal and devotion. Major Hindu holidays like Dashain and Tihar, which symbolize the victory of good over evil, include family gatherings, worship of the gods, and animal sacrifices. Buddhists celebrate Buddha Jayanti with peaceful prayers and lighting a candle procession, especially in places like Lumbini and Swayambhunath. Vibrant and rich in ancient traditions, indigenous groups such as the Newars celebrate unique holidays like Indra Jatra and Gai Jatra. These festivities protect Nepal's cultural heritage and encourage harmony among its people.</p>
+      <div id="making-a-table">
+        <table>
+          <caption>Table 1. Current Courses</caption>
+          <thead>
+          <tr>
+            <th scope="col" rowspan="2">Term</th>
+            <th scope="col" colspan="2">Course</th>
+          </tr>
+          <tr>
+            <th scope="col">Code</th>
+            <th scope="col">Name</th>
+            <th scope="col">Hrs/Week</th>
+          </tr>
+          </thead>
+          <tbody>
+          <tr>
+            <th scope="row" rowspan="7">Winter</th>
+          </tr>
+          <tr>
+            <td>CSD123</td>
+            <td>Computing Environment and Tools</td>
+            <td>3</td>
+          </tr>
+          <tr>
+            <td>CSD120</td>
+            <td>Communication</td>
+            <td>2</td>
+          </tr>
+          <tr>
+            <td>CSD110</td>
+            <td>Technology in Society</td>
+            <td>3</td>
+          </tr>
+          <tr>
+            <td>CSD113</td>
+            <td>Intro to Programming</td>
+            <td>4</td>
+          </tr>
+          <tr>
+            <td>CSD220</td>
+            <td>Hardware/OS/Network</td>
+            <td>3</td>
+          </tr>
+          <tr>
+            <td>CSD150</td>
+            <td>Computer Math</td>
+            <td>4</td>
+          </tr>
+          <tr>
+            <th scope="row" rowspan="7">Summer</th>
+          </tr>
+          <tr>
+            <td>CSD123</td>
+            <td>Programming Concepts</td>
+            <td>4</td>
+          </tr>
```

Comment:
> 1. `Lab 4 > Making a table > the table > body has > Fall Semester`
Summer / Spring also works here. Note that the early closing of the row is incorrect here.
```suggestion
          <tr>
            <th scope="row" rowspan="7">Summer</th>
            <td>CSD123</td>
            <td>Programming Concepts</td>
            <td>4</td>
          </tr>
```